src/main/java/seedu/address/storage/portmanager/PortManager.java

```diff
@@ -104,9 +104,18 @@ private Deck getImportedDeck(XmlExportableDeck targetDeck) throws DataConversion
 
     }
 
+    /**
+     * Convert the string into a file path.
+     * @param name The name of the file, can be the absolute or relative file path
+     * @return a Path that represents the file path
+     */
 
     private Path makeFilePath(String name) {
-        return basefilepath.resolve(name + ".xml");
+        if (name.substring(name.length() - 4).equals(".xml")) {
```

Comment:
I think we should change 4 to a constant so future readers know what the value represents.